company-policies/security.md

```diff
@@ -1,6 +1,5 @@
 ---
-title: Security Policy
-version: 1.1.6
```

Comment:
Looks like the extra colon causing a frontmatter error so it is showing this value on https://civicactions-handbook--1609.org.readthedocs.build/en/1609/company-policies/security/